handler/core/src/main/java/io/knotx/fragments/handler/debug/FragmentsDebugModeDecorator.java

```diff
@@ -0,0 +1,68 @@
+package io.knotx.fragments.handler.debug;
```

Comment:
Please add license header (why Travis build passed instead of rat plugin...).

Reply:
done